azure-pipelines.yml

```diff
@@ -154,6 +154,7 @@ jobs:
         displayName: yarn
 
       - script: |
+          yarn build --to @fluentui/babel-make-styles --no-cache
```

Comment:
Have to add this as otherwise Babel plugin is not ready to use...

Reply:
Please add a `displayName`

Reply:
Moved into a separate script with own `displayName`